### ISSUE TITLE
Add more clear instructions on serving the ACE in the workbench

### ISSUE THREAD

**Issue by rabwill** [SharePoint/sp-dev-docs#7313]
## Add more clear instructions on serving the ACE in the workbench

I ran `gulp serve -l --nobrowser` command as per instruction but did not get my webpart to show up in `https://{tenant}.sharepoint.com/_layouts/15/workbench.aspx`
I had to manually open `https://localhost:4321` and accept to proceed in my browser. Once this was done, the webpart showed up in the list of webparts in the local webserver.
This needs to be captured in the document.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 010f798c-ec19-8cc1-b9d3-84ca31cb6e62
* Version Independent ID: d76b36f9-620d-f82c-7122-7cd2a468dc6d
* Content: [Build your first SharePoint Adaptive Card Extension](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/viva/get-started/build-first-sharepoint-adaptive-card-extension)
* Content Source: [docs/spfx/viva/get-started/build-first-sharepoint-adaptive-card-extension.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/spfx/viva/get-started/build-first-sharepoint-adaptive-card-extension.md)
* Product: **sharepoint**
* Technology: **sharepoint-framework**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by waldekmastykarz** [comment]
Have you trusted the dev certificate as mentioned in the [setup instructions](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-development-environment#trusting-the-self-signed-developer-certificate)?

**Comment by rabwill** [comment]
@waldekmastykarz that makes sense with the assumption that the developer started this journey from 
[https://docs.microsoft.com/en-us/sharepoint/dev/spfx/viva/overview-viva-connections](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/viva/overview-viva-connections)
If they come to page [https://docs.microsoft.com/en-us/sharepoint/dev/spfx/viva/get-started/build-first-sharepoint-adaptive-card-extension](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/viva/get-started/build-first-sharepoint-adaptive-card-extension) to `Build your first SharePoint Adaptive Card Extension` would this be the case assuming that those folks are new to this? I felt like  had to jump to three links before I could start.
Would it help if we give minimum perquisites.
- Node 12 LTS
- Gulp
- Yoeman
And before the serve script, may be ask to trust the dev cetificate?
Happy to help too if this looks like an improvements.

**Comment by waldekmastykarz** [comment]
Good idea! What if linked to the existing setup steps and prerequisites so that we can keep them in one place and when they change we'd update them consistently throughout the docs?
